styles/gmail/catppuccin.user.css

```diff
@@ -78,18 +78,6 @@
     body {
       color: @text !important;
     }
-
-    /* input box */
-    .nr {
-      background-color: @surface1 !important;
-      color: @text !important;
-    }
-
-    .bs0 > .acM,
-    .bti > .btg {
-      color: @text !important;
-    }
```

Comment:
I think removing this has introduced a reggression here 
![image](https://github.com/catppuccin/userstyles/assets/71222764/e279d0d7-1bfa-4a09-8cdc-18c6dc8afe23)